Snake/SnakeTest/Laser_Test.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SnakeTest
+{
+    [TestClass]
+    public class Laser_Test
+    {
+        [TestMethod]
+        public void Move_Test_if_Laser_Moves_Down()
+        {
+
+            BusinessLayer.Laser testLaser = new BusinessLayer.Laser(2,2, BusinessLayer.Direction.Down);
+
+            testLaser.Move();
+
+            Assert.IsTrue(testLaser.returnX() == 3);
```

Comment:
this seems odd. the coordinates are 2,2 but the test looks for 3. something is not right